### PR TITLE
Clean forceQualifierUpdate files for doc bundles

### PR DESCRIPTION
**Pull request by MohananRahul**
Fix https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/2580

### CODE REVIEW

**Comment by akurtakov** [comment]
An extra commit removing the now not needed api filters is needed.

File: build/org.eclipse.pde.build.tests/test-sdk.product [hunk 1]
```diff
@@ -21,12 +21,14 @@
       </win>
    </launcher>
 
+   <plugins>
```
Comment (akurtakov) [inline build/org.eclipse.pde.build.tests/test-sdk.product:24]: This change is not needed.

File: build/org.eclipse.pde.build.tests/test-sdk.product [hunk 2]
```diff
@@ -21,12 +21,14 @@
...
+   </plugins>
+
    <features>
       <feature id="org.eclipse.platform"/>
       <feature id="org.eclipse.platform.source"/>
       <feature id="org.eclipse.jdt"/>
       <feature id="org.eclipse.jdt.source"/>
-      <!-- PDE must be installed as a root so it can be updated -->
```
Comment (akurtakov) [inline build/org.eclipse.pde.build.tests/test-sdk.product:29]: Please don't remove this comment.

**Comment by MohananRahul** [comment]
> An extra commit removing the now not needed api filters is needed.

Done

**Comment by akurtakov** [comment]
Thanks.

**Pull request merged**